opencog/nlp/relex2logic/det.scm

```diff
@@ -0,0 +1,59 @@
+
+(define det
+    (BindLink
+        (VariableList
+            (TypedVariableLink
+                (VariableNode "$a-parse")
+                (TypeNode "ParseNode")
+            )
+            (TypedVariableLink
+                (VariableNode "$noun")
+                (TypeNode "WordInstanceNode")
+            )
+            (TypedVariableLink
+                (VariableNode "$det")
+                (TypeNode "WordInstanceNode")
+            )
+        )
+        (AndLink
+            (WordInstanceLink
+                (VariableNode "$noun")
+                (VariableNode "$a-parse")
+            )
+            (WordInstanceLink
+                (VariableNode "$det")
+                (VariableNode "$a-parse")
+            )
+            (EvaluationLink
+                (DefinedLinguisticRelationshipNode "_det")
+                (ListLink
+                    (VariableNode "$noun")
+                    (VariableNode "$det")
+                )
+            )
+            (InheritanceLink 
+                (VariableNode "$noun")
```

Comment:
I think this rule is for the a second iteration write? In other words it is a second-level rule. Thus `nlp-fc` should be modified or the rule-base has to be modified into two and the output of the first layer piped to the second. 

@williampma @leungmanhin @bgoertzel  comment pls?